frontend/app/components/CardBack.tsx

```diff
@@ -46,8 +45,15 @@ export const CardBack = ({
         <Animated.View style={styles.divider} />
         <MeaningSection meaning={item.meaning} />
         <ExplanationSection explanation={item.explanation} />
-        <ExamplesSection examples={item.examples} />
+
+        <Text style={styles.sectionTitle}>Examples</Text>
```

Comment:
I see we're removing the flatlist, that makes a lot of sense since this is simpler, can we keep the examples component though? We will need that in the future anyway